covalent_slurm_plugin/slurm.py

```diff
@@ -62,9 +63,10 @@ class SlurmExecutor(BaseAsyncExecutor):
         address: Remote address or hostname of the Slurm login node.
         ssh_key_file: Private RSA key used to authenticate over SSH.
         remote_workdir: Working directory on the remote cluster.
+        options: Dictionary of parameters used to build a Slurm submit script.
         poll_freq: Frequency with which to poll a submitted job.
         cache_dir: Cache directory used by this executor for temporary files.
-        options: Dictionary of parameters used to build a Slurm submit script.
+        do_cleanup: Whether to perform cleanup or not on remote machine.
```

Comment:
can this be called just `cleanup` to keep it consistent with other executor plugins (like lambda) that aim to do the same thing?

Reply:
Might get confusing with the function name being `cleanup` as well. What is the function name in those plugins? In ssh it is `do_cleanup` - param and `cleanup` - function.

Reply:
I'd recommend to add the cleanup logic in `teardown` as that has been merged to trunk... so a `_cleanup` method is not required... but leave it up to your preference

Reply:
Added cleanup as part of teardown.